.gitignore

```diff
@@ -8,6 +8,7 @@ logs
 *.log
 npm-debug.log*
 .eslintcache
+.cspellcache
```

Comment:
We can also disable cache in cpsell. It doesn't differ much from no-cache

Reply:
It has a big changes when you have a lot of files, I don't think we need disable it, we use cache everywhere

Reply:
ok